src/pages/Home/index.tsx

```diff
@@ -9,6 +9,15 @@ export const Home = () => {
   const text = t("Welcome to The Home of Asylum Seekers");
 
   return (
+    <div style ={{
+      backgroundColor: "#ECF2FF",
+      maxHeight: "100vh",
+      minHeight: "100vh"
+    }}>
+    <Nav />
+      <div style={{ marginBottom: "20px", justifyContent: "center", alignItems: "center", height: "85vh", display: "flex", flexDirection: "column" }}>
```

Comment:
I think you may have missed adding the closing tags for these three divs you added here

Reply:
I can't seem to find what you are referring to? The div at line 21 should close at 29. The one at 22 closes on the same line. The one at line 23 should close at 28. The final div at line 24 should close at 27.

Reply:
Oh wait never mind. I had to pull the branch merge I did to this earlier. It seems that resolving an earlier conflict has messed up the code